### PR TITLE
docs: add kathara network plugin

### PR DESCRIPTION
**Pull request by dvdksn**
- docs(plugins): update Docker Store to Docker Hub
- docs(plugins): add Kathará engine network plugin
- docs(plugins): remove inline CSS markup
- docs(plugins): remove Weave engine network plugin

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by akerouanton** [comment]
Let me ping a few Kathará maintainers.

@lorenzo93 @glospoto @Skazza94 We want to add your project to our list of network plugins. Does that feel right to you? :slightly_smiling_face:

File: docs/extend/legacy_plugins.md [hunk 1]
```diff
@@ -42,10 +42,6 @@ Follow the instructions in the plugin's documentation.
 
 The sections below provide an inexhaustive overview of available plugins.
 
-<style>
-#DocumentationText  tr td:first-child { white-space: nowrap;}
-</style>
```
Comment (thaJeztah) [inline docs/extend/legacy_plugins.md:47]: Oh, this was a big hack indeed; did we have a replacement for this in the new layout? (preventing the column from wrapping?)

**Comment by Skazza94** [comment]
> Let me ping a few Kathará maintainers.
> 
> @lorenzo93 @glospoto @Skazza94 We want to add your project to our list of network plugins. Does that feel right to you? 🙂

Hi @akerouanton,
of course, it would be great for us and for the Kathará project! Thank you very much 😄

**Review by akerouanton — APPROVED** [review]
LGTM! :tada:

**Pull request merged**